WowPacketParser/Parsing/Parsers/CombatLogHandler.cs

```diff
@@ -213,6 +213,7 @@ private static void ReadSpellLogExecute(Packet packet, object index = null)
                             break;
                         }
                         case SpellEffect.DismissPet:
+                        case SpellEffect.CallPet:
```

Comment:
not there in 3.3.5

Reply:
The handler is identical for TBC and wotlk which is why I added it here. Should I clone the method instead for tbc?

Reply:
no, its actually fine like this since server would not even include a block for CALL_PET so this code wont be reached and read wrong